central/scannerdefinitions/handler/handler.go

```diff
@@ -478,7 +478,7 @@ func (h *httpHandler) startUpdaterAndOpenFile(u *requestedUpdater) (*os.File, ti
 	if u == nil {
 		return nil, time.Time{}, errors.New("Fail to initialize updater")
 	}
-	log.Info("Initializing updater")
+	log.Debug("Initializing updater")
```

Comment:
https://github.com/stackrox/stackrox/pull/9135 plans on just removing this. We can either wait for that to be merged (not sure when) or update this PR to just remove the log. Up to you

Reply:
@RTann  I'm fine closing this as long as the changes land within the next release, if that's planned to land in 4.4 I can close this PR.

Reply:
yes, it will be in the next release. right @daynewlee ?